packages/nodes-base/nodes/Postgres/Postgres.node.functions.ts

```diff
@@ -109,21 +114,36 @@ export async function pgUpdate(
 	const updateKey = getNodeParam('updateKey', 0) as string;
 	const columnString = getNodeParam('columns', 0) as string;
 
-	const columns = columnString.split(',').map(column => column.trim());
+	const updateColumnParts = updateKey.split(':');
+	const updateColumn = {
+		name: updateColumnParts[0],
+		cast: updateColumnParts[1],
+	};
```

Comment:
I think a destruction makes it more readable here 😊 
```suggestion
	const [updateColumnName, updateColumnCast] = updateKey.split(':');
	const updateColumn = {
		name: updateColumnName,
		cast: updateColumnCast,
	};
```